src/app/app.component.spec.ts

```diff
@@ -1,32 +1,48 @@
 import { TestBed, async } from '@angular/core/testing';
+import { APP_BASE_HREF } from '@angular/common';
 import { AppComponent } from './app.component';
-import { GameListComponent } from './game-list/game-list.component';
-import { CreateGameDialogComponent } from './create-game-dialog/create-game-dialog.component';
+import { GameListComponent } from './pages';
+import { ListedGameComponent } from './pages/game-list/listed-game/listed-game.component';
+import { CreateGameDialogComponent } from './pages/game-list/create-game-dialog/create-game-dialog.component';
+import { LoginScreenComponent } from './pages';
+
+import { RouterModule, Routes } from '@angular/router';
+
+const appRoutes: Routes = [
+  { path: 'login', component: LoginScreenComponent },
+  { path: 'game-list', component: GameListComponent },
+  {
+    path: '',
+    redirectTo: '/login',
+    pathMatch: 'full'
+  },
+  { path: '**', redirectTo: '/login' }
+];
 
 describe('AppComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [
         AppComponent,
         CreateGameDialogComponent,
-        GameListComponent
+        GameListComponent,
+        ListedGameComponent,
+        LoginScreenComponent
       ],
```

Comment:
Sorry for the ignorance but is one of these the lobby screen or are we missing that?